app/constants/account-codes.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  SFI_ARABLE_SOIL_INTRODUCTORY_AP: 'SOS273'
```

Comment:
Account codes go by line description/scheme combination rather than by the scheme code.

So a more accurate description here would be `SFI_GROSS_VALUE_AP`